Configuration/TCA/tx_variables_marker.php

```diff
@@ -136,6 +136,7 @@
                 'rows' => 4,
                 'eval' => 'required,trim'
             ],
+            'defaultExtras' => 'richtext:rte_transform[mode=css]',
```

Comment:
@xperseguers: Thanks for your response. I just adjusted that... and learnt something new